src/include/miopen/fusion/utils.hpp

```diff
@@ -84,6 +84,10 @@ inline bool WinoCommonIsApplicable(const FusionContext& context, const FusionDes
         return false;
     if(!conv_problem.IsFp32())
         return false;
+    if(conv_problem.HasNonPackedTensors())
+    {
+        return false;
+    }
```

Comment:
[Recommendation] Curly braces here and in other places may be omitted. The code in this case will look better when you need to check multiple conditions.

Reply:
I can remove them. My personal preference is to always have `{}` and have the starting `{` on the same line.